python/cudf/cudf/core/column_accessor.py

```diff
@@ -51,6 +51,9 @@ def from_zip(cls, data):
 
     def __getitem__(self, key):
         """Recursively apply dict.__getitem__ to get nested elements of d."""
+        # Don't reduce over characters in a single string key, just get.
+        if isinstance(key, str):
+            return super().__getitem__(key)
```

Comment:
When do we want to reduce? When there's a tuple? This feels somewhat brittle.

Reply:
Thanks for pointing this out. You're right, this should only be done for tuples (in particular, shouldn't be done for lists). I found the relevant pandas docs and will link in the comments.